docs/_partials/events/set_flash.rst

```diff
@@ -14,6 +14,41 @@ The :ref:`Crud Subject <crud-subject>` contains the following keys:
 All keys can be modified as you see fit, at the end of the event cycle they will be passed
 directly to ``SessionComponent::setFlash``.
 
-Defaults are stored in the ``messages`` configuration array for each action.
+Defaults are stored in the ``messages`` configuration array for each :doc:`action </actions>`.
 
 If you do not want to use this feature, simply stop the event by calling it's ``stopPropagation()`` method.
+
+If you'd like to customise the flash messages that are used, perhaps you're using
+`friendsofcake/bootstrap-ui <https://github.com/friendsofcake/bootstrap-ui>`_. It's actually quite simple to do, and can
+be done as part of the component configuration or on the fly.
+
+.. code-block:: phpinline
+
+  public function initialize()
+  {
+        $this->loadComponent('Crud.Crud', [
+            'actions' => [
+                'edit' => [
+                    'className' => 'Crud.Edit',
+                    'messages' => [
+                        'success' => [
+                            'params' => ['class' => 'alert alert-success alert-dismissible']
+                        ],
+                        'error' => [
+                            'params' => ['class' => 'alert alert-danger alert-dismissible']
+                        ]
+                    ],
+                ]
+            ]
+        ]);
+  // etc
```

Comment:
Yeah, the `//etc` was indicative of there being more code in the `initialize` method, but I'll add the closing brace as you're right, it will look neater and be more copy and pastable.